client/src/components/App.jsx

```diff
@@ -99,17 +99,33 @@ class App extends React.Component {
           price = data.listPrice.split('.')[0];
         }
         const tags = [];
-        if (data.isNew) {
-          tags.push('NEW');
+        if (country == 'caFR') {
```

Comment:
Strict equality (===) is safer to use (prevent type coercion: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality)